### ISSUE TITLE
Error in stackblitz for Dynamic component loader

### ISSUE THREAD

**Issue by bastienmoulia** [angular/angular#48047]
### Describe the problem that you experienced

When opening the stackblitz exemple for Dynamic component loader there is a TS error.

```
Error: src/app/ad-banner.component.ts:45:5 - error TS2322: Type 'Timer' is not assignable to type 'number'.

45     this.interval = setInterval(() => {
       ~~~~~~~~~~~~~
```

Because `setInterval` is from `NodeJS` and not `window`.

### Enter the URL of the topic with the problem

https://angular.io/generated/live-examples/dynamic-component-loader/stackblitz.html

### Describe what you were looking for in the documentation

Opening the page https://angular.io/guide/dynamic-component-loader

### Describe the actions that led you to experience the problem

Clicking the link *live exemple*

### Describe what you want to experience that would fix the problem

```
this.interval = window.setInterval(() => {
  this.loadComponent();
}, 3000);
```

**Comment by JoostK** [comment]
This broke due to https://github.com/ReactiveX/rxjs/commit/fce9aa12931796892673581761bba1f7ceafabff in rxjs 7.5.6, which is causing Node types to be pulled into application compilations.

**Comment by JeanMeche** [comment]
I believe this has been fixed by ReactiveX/rxjs@c1a07b7 and shipped in RxJS 7.6 !

**Comment by JeanMeche** [comment]
The issue can be closed, RxJS has been updated and the examples.
